common/test/acceptance/tests/discussion/test_discussion_management.py

```diff
@@ -271,3 +288,163 @@ def test_verify_that_deselecting_the_final_child_deselects_category(self):
+        self.verify_save_confirmation_message(self.scheme_key)
+        self.check_discussion_topic_visibility(visible=False)
+
+        # Reload the page and make sure that the change was persisted
+        self.reload_page(topics_visible=False)
+        self.assertTrue(self.discussion_management_page.division_scheme_visible(
+            self.discussion_management_page.COHORT_SCHEME)
+        )
+        self.assertEqual(
+            self.discussion_management_page.NOT_DIVIDED_SCHEME,
+            self.discussion_management_page.get_selected_scheme()
+        )
+
+        # Select "cohort" again and make sure that the discussion topics appear.
+        self.discussion_management_page.select_division_scheme(self.discussion_management_page.COHORT_SCHEME)
+        self.verify_save_confirmation_message(self.scheme_key)
+        self.check_discussion_topic_visibility(visible=True)
+
+    def test_disabling_cohorts(self):
+        """
+        Test that disabling cohorts hides the cohort division scheme iff it is not the selected scheme
+        (even without reloading the page).
+        """
+        # TODO: will be added as part of AJ's work.
+        pass
+
+    def test_enrollment_track_scheme_not_present(self):
+        """
+        Test that the enrollment track scheme is not visible if there is a single enrollment mode.
+        """
+        self.add_enrollment_modes(['audit'])
+        self.assertFalse(
+            self.discussion_management_page.division_scheme_visible(
+                self.discussion_management_page.ENROLLMENT_TRACK_SCHEME
+            )
+        )
+
+    def test_enrollment_track_scheme_persisted(self):
+        """
+        Test that the enrollment track scheme is visible if there are multiple enrollment tracks,
+        and that the selection can be persisted.
+
+        Also verifies that the cohort division scheme is not presented if cohorts are disabled and cohorts
+        are not the selected division scheme.
+        """
+        self.add_enrollment_modes(['audit', 'verified'])
+        self.assertTrue(
+            self.discussion_management_page.division_scheme_visible(
+                self.discussion_management_page.ENROLLMENT_TRACK_SCHEME
+            )
+        )
+        # And the cohort scheme is initially visible because it is selected (and cohorts are enabled).
+        self.assertTrue(
+            self.discussion_management_page.division_scheme_visible(self.discussion_management_page.COHORT_SCHEME)
+        )
+
+        self.discussion_management_page.select_division_scheme(self.discussion_management_page.ENROLLMENT_TRACK_SCHEME)
+        self.verify_save_confirmation_message(self.scheme_key)
+        self.check_discussion_topic_visibility(visible=True)
+
+        # Also disable cohorts so we can verify that the cohort scheme choice goes away.
+        self.disable_cohorting(self.course_fixture)
```

Comment:
I also created a stub test point for you, but we can remove it in your PR if it is not useful.